### DISCUSSION TITLE
Snap version seems to be outdated

### DISCUSSION THREAD

**Discussion by technarf** [zwave-js/zwave-js-ui#2363] [Q&A] [answered]
Hi, 
I'm using zwavejs2mqtt not for so long now, but it seems the snap version is outdated. 
When I run `snap info zwavejs2mqtt` I obtain this result :
`tracking:     latest/stable
refresh-date: 18 days ago, at 22:21 CET
channels:
  latest/stable:    v6.5.2 2022-03-23 (344) 137MB -
  latest/candidate: v6.6.0 2022-03-24 (351) 138MB -
  latest/beta:      ↑                             
  latest/edge:      v6.6.2 2022-03-25 (355) 137MB -
installed:          v6.5.2            (344) 137MB -`
So the latest stable version in snap system is 6.5.2 so there are 4 versions since this one (the current one when this message is written is 6.7.0).
How can I do to have a more up to date version without switching to beta or "something like that" channels ?
Thanks

**Comment by robertsLando** [comment]
@jmgiaever

**Reply by jmgiaever** [reply]
Hi, I'm on a holiday for 9 more days and since I don't have access to my network I didn't want to version bump to a major new version now.

If someone is on the edge channel and can test, then I can do it.

**Comment by jmgiaever** [comment]
Hi. Read the information in `snap info zwavejs2mqtt`. If you want a rolling release cycle, switch to `latest/candidate`.

**Comment by jmgiaever** [comment, marked as answer]
Latest (`6.7.4`) is now in the `latest/candidate` channel. This is the channel that is used for the rolling releases. It will become available in the `latest/stable` channel when the latest patch release of `6.7.x` version has landed - that is when `6.8.x` is here. 

Anyone who wants the rolling release with the latest must use the `candidate`

```
sudo snap switch zwavejs2mqtt --channel=latest/candidate
```

**Reply by jmgiaever** [reply]
@robertsLando you can close this

**Reply by robertsLando** [reply]
Cannot close this as it's  a question but I have marked your comment as answer :) Thanks @jmgiaever

**Reply by jmgiaever** [reply]
Oh, sorry. Didn't notice. Thanks 😅
